s3/compress.go

```diff
@@ -101,16 +99,17 @@ func (c *Compressor) zipExport(ctx context.Context, prefix, filename, s3key stri
 
 		fileMeta = append(fileMeta, *tempFileMeta)
 
-		header, err := tar.FileInfoHeader(fi, basename)
+		header, err := zip.FileInfoHeader(fi)
 		if err != nil {
 			return fmt.Errorf("failed to create file header: %w", err)
 		}
 		header.Name = basename
 
-		if err = tarWriter.WriteHeader(header); err != nil {
+		var zippedFile io.Writer
```

Comment:
This should be defined at the top for consistent style.

Reply:
I want to limit the scope of that variable so I am creating/instantiating it as far down as possible (to just where it is used).